index.html

```diff
@@ -12,7 +12,14 @@
             <nav>
                 <section class="LargeSalmon">
                     <img src= "C:\Users\Ray\codefellows\201\cookielab\cookie-stand\salmon.png" alt="Photograph of large odd salmon" title= "large salmon">
```

Comment:
If you open this page anywhere except on your computer, none of these links work currently because they are absolute links. You need to make these relative links.